text/3-entrees/basic-pow.md

```diff
@@ -0,0 +1,50 @@
+# Basic Proof of Work
+*[`nodes/basic-pow`](https://github.com/substrate-developer-hub/recipes/tree/master/nodes/basic-pow)*
+
+The `basic-pow` node uses a minimal [Proof of Work](https://en.wikipedia.org/wiki/Proof_of_work) consensus engine to reach agreement over the blockchain. Being our first recipe on consensus, this node is kept intentionally simple. It omits some features that would make it practical for real-world use such as difficulty adjustment and block rewards. Nonetheless, it is a real usable consensus engine that will teach us many useful aspects of dealing with consensus and prepare us to understand more advanced consensus engines in the future. In particular we will learn about:
+* Substrate's [`BlockImport` trait](https://substrate.dev/rustdocs/master/sp_consensus/block_import/trait.BlockImport.html)
+* Substrate's [import pipeline](https://substrate.dev/rustdocs/master/sp_consensus/import_queue/index.html)
+* Structure of a typical [Substrate Service](https://substrate.dev/rustdocs/master/sc_service/index.html)
+* Configuring [`InherentDataProvider`](https://substrate.dev/rustdocs/master/sp_authorship/struct.InherentDataProvider.html)s
+
+## The Structure of a Node
+
+You may remember from the [hello-substrate recipe](../2-appetizers/1-hello-substrate.md) that a Substrate node has two parts. An outer part that is responsible for gossiping transactions and blocks, handling [rpc requests](./custom-rpc.md), and reaching consensus. And a runtime that is responsible for the business logic of the chain. This architecture diagram illustrates the distinction.
+
+![Substrate Architecture Diagram](../img/substrate-architecture.png)
+
+In principle the consensus engine, part of the outer node, is agnostic over the runtime that is used with it. But in practice, most consensus engines will require the runtime to provide certain [runtime APIs](./runtime-api.md) that effect the engine. For example Aura, and Babe, query the runtime for the set of validators. A more real-world PoW consensus would query the runtime for the bock difficulty. Luckily this simple PoW engine really is agnostic over the runtime, and thus we will use the familiar `super-runtime`.
```

Comment:
@rphmeier This is my understanding of why the bae and grandpa runtime APIs would be harmless.

Reply:
Yes, having extra unused runtime APIs is not a problem.

Also, it can go vice-versa. The consensus engine can also expect that the runtime does things like issuing digests with a specific format at certain times. The consensus engine won't behave correctly if they aren't issued, but it can ignore any extra (from its perspective) digests.

And typo: `effect` should be `affect`.